src/content/events/2024/bytesize_blog.md

```diff
@@ -0,0 +1,22 @@
+---
+title: 'Bytesize: nf-core blog'
+subtitle: Matthias Hörtenhuber, SciLifeLab Data Centre
+type: talk
+start_date: '2024-01-23'
+start_time: '13:00+02:00'
+end_date: '2024-01-23'
+end_time: '13:00+02:00'
+location_url: https://kth-se.zoom.us/j/68390542812
+---
+
+# nf-core/bytesize
+
+Join us for our **weekly series** of short talks: **“nf-core/bytesize”**.
+
+Just **15 minutes** + questions, we will be focussing on topics about using and developing nf-core pipelines.
+These will be recorded and made available at <https://nf-co.re>
+It is our hope that these talks / videos will build an archive of training material that can complement our documentation. Got an idea for a talk? Let us know on the [`#bytesize`](https://nfcore.slack.com/channels/bytesize) Slack channel!
+
+## Bytesize: nf-core blog
+
+Since the 12th of January 2024, we have an official nf-core blog on our nf-core website. While it will be used by the core team and developers to update the community about ongoing project, it is also a great way for you to get active!. Matthias Hörtenhuber ([@mashehu](https://github.com/mashehu)) will introduce us to the vision for the brand new and shiny blog, which you can access [here](https://nf-co.re/blog)
```

Comment:
```suggestion
Since the 12th of January 2024, we have an official nf-core blog at [nf-co.re/blog](https://nf-co.re/blog). While it will be used by the core team and developers to update the community about ongoing project, it is also a great way for you to get active!. Matthias Hörtenhuber ([@mashehu](https://github.com/mashehu)) will introduce us to the vision for the brand new and shiny blog and how you can contribute to it.)
```